frontend/src/main/java/starter/project/frontend/CarController.java

```diff
@@ -4,72 +4,62 @@
 import org.springframework.web.bind.annotation.*;
 import org.springframework.web.server.ResponseStatusException;
 import starter.project.domain.Car;
+import starter.project.domain.Service;
+
 import java.util.*;
 
 @RestController
 public class CarController {
-   private Map<Integer, Car> map;
```

Comment:
you've completely changed this endpoint

Reply:
i didnt think private Map<Integer, Car> map; still had a purpose as the class wouldn't be using it store the data anymore